.gitignore

```diff
@@ -2,4 +2,5 @@
 .vscode/
 .vs/
 .DS_Store
-build/
\ No newline at end of file
+build/
+week2/part4/CMakeFiles/
```

Comment:
Let's change this to `**/CMakeFiles/`.

This will match any directory named "CMakeFiles" at any level in your project directory and ignore its contents. The double asterisks (**) represent any level of directories.

Reply:
I made this change.